.github/workflows/trivy-pr-check.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Trivy Scan'
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0005   
**Check Name:** persist-credentials is true   
**Severity:** HIGH   
**Message:** Consider adding persist-credentials: false to the checkout action in job 'Build' inputs  
**Resolution:** Add persist-credentials: false to the checkout action  
[This comment was created by Aqua Pipeline]